logic/src/commonMain/kotlin/com/wire/kalium/logic/data/auth/login/ProxyCredentialsModel.kt

```diff
@@ -0,0 +1,3 @@
+package com.wire.kalium.logic.data.auth.login
+
+data class ProxyCredentialsModel(val username: String?, val password: String?)
```

Comment:
How about deleting this class and move the properties to ProxyCredentials.UsernameAndPassword

Reply:
I was passing pair and after that pass the user name and pass to the ProxyCredentials.UsernameAndPassword
but I saw this as more organized way to pass it to the mapper and the scope , better than pass pair or pass the username and password params

Reply:
Then let's rename this to simply `ProxyCredentials` and the sealed class can be called something like `ProxyAuthentication`